.origintrail_noderc.image

```diff
@@ -0,0 +1,38 @@
+{  
```

Comment:
Please state only values different to the default configuration. Avoid repeating same values.